INSTALL.md

```diff
@@ -16,6 +16,18 @@ export PKG_CONFIG_PATH="_OpenSSL_prefix_/lib/pkgconfig:$PKG_CONFIG_PATH"
 
 ## OS-specific setup for building Scilla
 
+<details><summary>vcpkg</summary>
+
+- Clone vcpkg to a separate location:
+
+   ```shell
+   $ git clone https://github.com/Microsoft/vcpkg.git /path/to/vcpkg
+   $ cd /path/to/vcpkg && git checkout 2022.07.25 && ./bootstrap-vcpkg.sh
+   $ cd /path/to/zilliqa
+   $ export VCPKG_ROOT=/path/to/vcpkg
```

Comment:
This I'm not happy with, as it suggests that it is now impossible to build Scilla without first building Zilliqa. Is there no way to build Scilla independently when using vcpkg?